internals/controllers/migrationhierarchy_controller.go

```diff
@@ -138,3 +155,14 @@ func (r *MigrationHierarchyReconciler) SetupWithManager(mgr ctrl.Manager) error
 		For(&danav1.MigrationHierarchy{}).
 		Complete(r)
 }
+
+// addSnsToSnsEvent takes two paramaters: snsName and snsNamespace
+// then add the sns to the sns event channel for trigger new resync for the sns
```

Comment:
`// addSnsToSnsEvent takes two paramaters: snsName and snsNamespace
// then adds the sns to the sns event channel to trigger new resync for the sns